modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/misc/Git.kt

```diff
@@ -6,10 +6,10 @@ import com.google.inject.Inject
 import java.io.File
 
 class Git @Inject constructor() {
-    fun maybeTagRelease(project: Project, uploadResult: TaskResult, autoGitTag: Boolean) : TaskResult {
+    fun maybeTagRelease(project: Project, uploadResult: TaskResult, autoGitTag: Triple<Boolean, String, String>) : TaskResult {
```

Comment:
Yes. Instead of going crazy with variable arguments, I'd rather have the older version `autoGitTag(Boolean)` and an overloaded one with `autoGitTag(AutoGitTagConfig)`.

Makes sense?